### ISSUE TITLE
how to restore a snapshoted application

### ISSUE THREAD

**Issue by oupala** [ecarre-dockerfiles/rhc#1]
I have a snapshot of an application, generated by this command:

``` bash
ssh 558d47bb3fffbd000287@myapp-mydomain.mybroker 'snapshot' > myapp.tar.gz
```

I'd like to restore it to another application, and I get the following error:

``` bash
rhc snapshot restore -a anotherapp -f myapp.tar.gz
Archive not found: myapp.tar.gz
```

The archive `myapp.tar.gz` **is** in the local directory. This issue might be related to the fact that `rhc` runs in a container.

Note that the `rhc git-clone` command does not work either. This might be the same issue.

**Comment by emmanuelCarre** [comment]
Hi,

Sorry for delay...I think, you have found a solution or workaroud.

For missing archive, you have to use an volume between your host and your container (see more on docker documentation [here](https://docs.docker.com/engine/userguide/containers/dockervolumes/)). Be carefull if you use an VM like boot2docker (windows and Mac), may be you have to share folder between host => VM => container.

I never use command rhc git-clone. I always use native git and multi remote feature to manage my apps. Do you have the trace for this error?
I will try it.

**Comment by oupala** [comment]
I have not found a workaround since I had my problem, except running everything out of docker. Which is not a convenient solution.

For `rhc git-clone`, see #2.

**Comment by oupala** [comment]
Note that this issue will be resolved once issue #2 is resolved also.

The change explained in [this comment](/ecarre-dockerfiles/rhc/issues/2#issuecomment-243140751) fixes all problem related to local files management. git-clone, snapshot and restore were all having the same issue.
